src/roblox_api/cframe.rs

```diff
@@ -0,0 +1,112 @@
+use rlua::{UserData, UserDataMethods, Value as LuaValue};
+
+use crate::value::{CFrameValue, Vector3Value};
+
+pub struct CFrame;
+
+impl CFrame {
+    fn from_position(x: f32, y: f32, z: f32) -> CFrameValue {
+        CFrameValue::new(rbx_dom_weak::types::CFrame::new(
+            rbx_dom_weak::types::Vector3::new(x as f32, y as f32, z as f32),
+            // TODO: replace with `rbx_dom_weak::types::Matrix3::identity()` once
+            // a version higher than 0.3.0 of rbx_types ships
+            rbx_dom_weak::types::Matrix3::new(
```

Comment:
Please import these types instead of fully qualifying them. It makes this code easier to read!